modules/currency_denomination/commerce_pos_currency_denomination.module

```diff
@@ -0,0 +1,195 @@
+  $items = array();
+
+  $items['admin/commerce/config/pos/denomination'] = array(
+    'title' => 'Configure Currency Denominations',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('commerce_pos_currency_denomination_settings_form'),
+    'access callback' => '_commerce_pos_access_check',
+    'access arguments' => array('process commerce pos sales'),
+    'file' => 'includes/commerce_pos_currency_denomination.admin.inc',
+  );
+
+  $items['admin/commerce/config/pos/denomination-type/add'] = array(
+    'title' => 'Add Currency Denomination Type',
+    'page callback' => 'drupal_get_form',
+    'type' => MENU_CALLBACK,
+    'page arguments' => array('commerce_pos_currency_add_denomination_type_form'),
+    'access callback' => '_commerce_pos_access_check',
+    'access arguments' => array('process commerce pos sales'),
+    'file' => 'includes/commerce_pos_currency_denomination.admin.inc',
+  );
+
+  $items['admin/commerce/config/pos/denomination-type/%/delete'] = array(
+    'title' => 'Delete Currency Denomination Type',
+    'page callback' => 'drupal_get_form',
+    'type' => MENU_CALLBACK,
+    'page arguments' => array('commerce_pos_currency_delete_denomination_type_form', 5),
+    'access callback' => '_commerce_pos_access_check',
+    'access arguments' => array('process commerce pos sales'),
+    'file' => 'includes/commerce_pos_currency_denomination.admin.inc',
+  );
+
+  $items['admin/commerce/config/pos/denomination-type/%/denomination/add'] = array(
+    'title' => 'Add Denomination',
+    'page callback' => 'drupal_get_form',
+    'type' => MENU_CALLBACK,
+    'page arguments' => array('commerce_pos_currency_add_denomination_form', 5),
+    'access callback' => '_commerce_pos_access_check',
+    'access arguments' => array('process commerce pos sales'),
+    'file' => 'includes/commerce_pos_currency_denomination.admin.inc',
+  );
+
+  $items['admin/commerce/config/pos/denomination-type/%/denomination/view'] = array(
+    'title' => 'View Denominations',
+    'page callback' => 'commerce_pos_currency_view_denominations',
+    'type' => MENU_CALLBACK,
+    'page arguments' => array(5),
+    'access callback' => '_commerce_pos_access_check',
+    'access arguments' => array('process commerce pos sales'),
+    'file' => 'includes/commerce_pos_currency_denomination.admin.inc',
+  );
+
+  $items['admin/commerce/config/pos/denomination-type/%/denomination/%/delete'] = array(
+    'title' => 'Delete Denomination',
+    'page callback' => 'drupal_get_form',
+    'type' => MENU_CALLBACK,
+    'page arguments' => array('commerce_pos_currency_delete_denomination_form', 5, 7),
+    'access callback' => '_commerce_pos_access_check',
+    'access arguments' => array('process commerce pos sales'),
+    'file' => 'includes/commerce_pos_currency_denomination.admin.inc',
+  );
+
+  return $items;
+}
+
+/**
+ * Returns all the denominations from the db for a currency code.
+ *
+ * @param array $currency_codes

```

Comment:
There should be a space between params and return. Also should provide a brief description of the param. https://www.drupal.org/docs/develop/coding-standards/api-documentation-and-comment-standards#param